op_crates/fetch/26_fetch.js

```diff
@@ -1,5 +1,14 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
+// @ts-check
```

Comment:
What does this do?

Reply:
Makes my VS Code scream at me if I use the wrong type in the wrong spot. We don't actually run TSC on the code in CI. Purely for development experience